secrets/fake/fake.go

```diff
@@ -0,0 +1,40 @@
+package fake
+
+import (
+	"errors"
+	"github.com/libopenstorage/openstorage/secrets"
+)
+
+var (
+	// ErrNotImplemented default secrets in OSD
+	ErrNotImplemented = errors.New("Not Implemented")
+)
+
+type Fake struct {
```

Comment:
nit: can we call this DefaultSecretManager instead Fake?
Fake sounds like a testing object; like stubs and mocks

Reply:
may be call it as a NullSecretManager.

Reply:
so package structure should also change, secrets/fake/fake.go to say secrets/defaultsecretmanager(or nullsecretmanager). How about secrets/manager/defaultmanager.go?